bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -678,6 +678,9 @@ private String getStatusMessage(PullRequestCheckIssueVisitor visitor,
                 progressBody.append("s");
             }
             progressBody.append("\n");
+
+            var existingRequested = false;
```

Comment:
This variable name confuses me. I would suggest `requestPresent`.

Reply:
Sure